src/config.ts

```diff
@@ -261,18 +261,11 @@ export async function getGlobalGithubConfig(
     }
 
     if (remoteUrl?.source === 'github.com') {
-      const gitRoot = await git.revparse(['--show-toplevel']);
-      const projectPath = path.posix.format(
-        path.parse(path.relative(gitRoot, configDir))
-      );
       repoGithubConfig = {
         owner: remoteUrl.owner,
         repo: remoteUrl.name,
-        projectPath,
       };
     }
-  } else if (!repoGithubConfig.projectPath) {
-    repoGithubConfig.projectPath = '.';
```

Comment:
Still finding some other mentions of `projectPath`. Remove as well?

```
$ ag projectPath
README.md
287:`projectPath`. Although Craft can infer the relative path of your project, it
294:  projectPath: sentry-rails

CHANGELOG.md
23:- feat: Add projectPath config option to GitHub (#220)

src/schemas/project_config.ts
29:  projectPath?: string;

src/schemas/projectConfig.schema.ts
23:        projectPath: {
```

Reply:
The ones under schemas should stay to Craft not error out when this value is set. I'll remove the Readme reference.

Reply:
Bonus points: Comment in config schema that this is vestigial for backwards compatibility.